README.md

```diff
@@ -22,66 +22,84 @@ Use one API, thanks to Segment.io's [analytics.js](https://segment.com/docs/libr
 - [Contributing](#contributing)
 - [License](#license)
 
-## Install
+## Background
+
+### Auto Analytics 2.0+
+With version 2.X we are **removing the embedded Segment `analytics.js` module**. With version 2.0.0 and beyond you will need to build your own Segment `analytics.js` module _**manually**_.
+
+**NOTE:** _We have **not** added the package Segment `analytics.js` as an explicit dependency_ because some developers will prefer to add only individual integrations or create a customized `analytics.js` module in order to keep the weight of the package to a minimum.
+
+### Auto Analytics 1.0+
+With the first version of this package we basically extracted code from our more Meteor-specific Atmosphere package to create a more generally usable package for the whole JavaScript community.
+
+
+## Quick Start
 
 ```sh
-npm install @okgrow/auto-analytics --save
+npm install --save @okgrow/auto-analytics
 ```
 
-## Currently Supported Analytic Services
+```js
+import OKGAnalytics from '@okgrow/auto-analytics';
+// NOTE: You must build your own analytics.js, see Creating Segment's analytics.js for details.
+import analytics from './your-custom-build/analytics.min.js';
 
-* Amplitude
-* Chartbeat
-* comScore
-* Google Analytics
-* HubSpot
-* Intercom
-* Keen IO
-* KISSmetrics
-* Mixpanel
-* Quantcast
-* Segment.io
+// Add your analytics integrations and their tracking ids + config options here.
+const settings = {
+  analytics,
+  integrations: {
+    'Google Analytics': { 'trackingId': 'Your tracking ID' },
+    'Mixpanel': { 'token': '...', 'people': true },
+  },
+  options: {
+    // Segment options to be passed to initialize() from analytics.js-core
+  },
+  autorun: true, // Defaults to true if not provided.
+};
```

Comment:
This looks like a slightly different structure from what I see on CHANGELOG.md:10. There I see an object pass which will have two keys/properties: `analytics` (the analytics object) and `settings` (which I presume would _contain_ the three other keys/properties I see here: `integrations`, `options` and `autorun`.)

Reply:
@ccuilla Thanks for spotting, have fixed the `changelog`.